File: VonexDealer.API/Models/Order/Landline.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using VonexDealer.API.Repository;

namespace VonexDealer.API.Models.Order
{
    public class Landline
    {
        public Int64 LandlineID { get; set; }
        public Int64 OrderID { get; set; }
        //public List<Address> Addresses { get; set; }

        public List<Churn> Churns { get; set; }
        public List<NewPSTN> NewServices { get; set; }
        public List<ISDN> ISDNs { get; set; }

        public static implicit operator Landline(LandlineRepository v)
        {
            throw new NotImplementedException();
        }
    }
}

File: VonexDealer.Vue/ClientApp/models/Landline.js
export class Landline {
    constructor({
        landlineID = 0,
        orderID = 0,
        churns = [],
        newServices = [],
        isdNs = []
    } = {}) {
        this.landlineID = landlineID;
        this.orderID = orderID;
        this.churns = churns;
        this.newServices = newServices;
        this.isdNs = isdNs;
    }

}
export function createLandline(data) {
    return Object.freeze(new Landline(data));
}
